backend/src/utils/sentry.js

```diff
@@ -0,0 +1,17 @@
+import * as Sentry from '@sentry/node'
+
+export function captureExceptionWithUser(user, fn) {
+  Sentry.withScope(function(scope) {
+    scope.setTag('username', user.get('username'))
+    scope.setExtra('user', {
+      email: user.get('email'),
+      roles: user.getGroupNames(),
+      isAdsyUser: user.isAdsyUser(),
+      isAdmin: user.isEmployee()
```

Comment:
Wouldn't this have to be negated? `!user.isEmployee`

Reply:
Fixed it